python/whylogs/api/logger/segment_processing.py

```diff
@@ -129,15 +133,19 @@ def _log_segment(
     pandas: Optional[pd.DataFrame] = None,
     row: Optional[Mapping[str, Any]] = None,
     segment_cache: Optional[SegmentCache] = None,
+    segment_key_values: Optional[Dict[str, str]] = None,
 ) -> Dict[Segment, Any]:
     segments: Dict[Segment, Any] = {}
     pandas, row = _pandas_or_dict(obj, pandas, row)
     if partition.filter:
+        # TODO: segment_key_values don't apply to filter since they're explicitely added
```

Comment:
Should verify this assumption

Reply:
The comment looks correct, can remove the TODO part of it